### ISSUE TITLE
Compute PBO (probability of backtest overfitting) and Deflated Sharpe for our real studies — the data is already on disk

### ISSUE THREAD

**Issue by mulhamfetna** [mulhamfetna/trading-strategy-finder#106]
Child of #103.

Two standard instruments from the same literature, both computable from data we already have, neither ever applied here.

## 1. PBO via CSCV

Bailey, Borwein, López de Prado & Zhu, *The Probability of Backtest Overfitting*, J. Computational Finance 20(4), 39–69. [PDF](https://www.davidhbailey.com/dhbpapers/backtest-prob.pdf)

**PBO** = the probability that the configuration selected best in-sample lands **below the median out-of-sample**.

Method (CSCV — combinatorially symmetric cross-validation):
- build the `T × N` performance matrix (T time observations, N trial configurations)
- split rows into `S` disjoint blocks; form all `C(S, S/2)` train/test combinations (S=16 → 12,780)
- per combination: pick the in-sample winner, find its out-of-sample rank, take the logit `λ = ln(ω̄/(1−ω̄))`
- `PBO = ∫₋∞⁰ f(λ) dλ` — the rate at which in-sample winners underperform the OOS median

**Customary rejection threshold: PBO > 0.05.** The authors' own examples: a synthetic overfit case scores **74%**; a real strategy scores **0.04%**.

## 2. Deflated Sharpe Ratio

Bailey & López de Prado, *The Deflated Sharpe Ratio*, JPM 40(5), 94–107. [PDF](https://www.davidhbailey.com/dhbpapers/deflated-sharpe.pdf)

```
SR₀  = √V[{SR̂ₙ}] · ( (1−γ)Z⁻¹(1−1/N) + γZ⁻¹(1−e⁻¹/N) )
DSR = Z[ (SR̂ − SR₀)·√(T−1) / √(1 − γ₃·SR̂ + ((γ₄−1)/4)·SR̂²) ]
```

Needs: `N` (independent trials — from the sibling issue), `V[{SR̂ₙ}]` (variance across trial Sharpes), `T`, skew `γ₃`, kurtosis `γ₄` of the selected strategy's returns.

**Particularly relevant to us:** our per-trade distribution is known to be fat-tailed (±$1,600 tail), and the DSR penalises exactly that through `γ₃`/`γ₄`. Their worked example: an annualised Sharpe of **2.5 over 5 years** fails 95% confidence at `N = 100` and would have passed at `N = 46`.

## Scope

Apply both to:
- at least 3 completed Optuna studies of different sizes
- the **deployed champions** (`best_*`), which have never been deflated

## What a bad result means

If PBO is high or DSR is below threshold for the deployed set, that does **not** automatically invalidate the book — the champions have separate OOS checks and #87 already flags the short history. It means the *selection procedure* is producing results the sample cannot support, and it changes what "a better search" is worth.

Report the numbers plainly either way.

Refs #103, #87, #90

**Comment by mulhamfetna** [comment]
**SYSTEM-CONTEXT UPDATE (state pin 2026-08-20, commit 98f5099, ledger 52/52 both machines).** Since this issue's last activity: WS-FUSION ran and CLOSED (14 pre-registered studies; the power-forecast layer deployed v5.4.3; the fused vol forecast won its quality stage; every P&L consumer closed with a law; state-blind entry / flat sizing / frozen geometry are now MEASURED choices). Pinned income statement: box ≈$840k/yr 2026-OOS + news $67,767/window 2024→26 (≈$1.167M/window model-grade at approved tiers), ledger 52/52, paper-only. **This optimizer-arm issue is UNCHANGED by the fusion era** — its queue position stands; the next workstream is the WS-EARN return. Current state of record: `docs/PROGRESS-RECORD.md` + `docs/SYSTEM-LAYERS-ANALYSIS.md` (§5 = per-layer profit attribution).

**Comment by mulhamfetna** [comment]
**ARCHITECTURE UPDATE — v5.5.0 RELEASED (2026-08-20; ledger 59/59 both machines).** The system now runs THREE deployed forecast components on TWO rails: the tape rail (the HAR-RV gate inside every champion) and the calendar rail (the power layer v5.4.3 + the NEW two-calendar forecast layer E-D1 — the gate's nightly blindness schedule, ROUTING the two certified repairs; routing-never-fitting is the architecture law, interference CI-proven). Era 10 complete (7 studies): earnings power forecastable (ρ .46), the blindness law spans both calendars, the 924-row substrate frozen, state-blind extended to SIZE, three refused near-misses machine-verified. Income structure unchanged: box ≈$840k/yr + news $67,767/window (≈$1.167M model-grade), calendar layers $0-by-design. State of record: `SYSTEM-LAYERS-ANALYSIS.md` (v5.5.0) + `PROGRESS-RECORD.md`. **Phase 3 (×news×indicators) initiating.** **This optimizer-arm issue is unchanged by v5.5.0** — queue position stands.

**Comment by mulhamfetna** [comment]
**ARCHITECTURE PIN — v5.5.1, THE ROADMAP COMPLETE (2026-08-20; ledger 64/64 both machines).** The architecture of record: **two income engines** (box ≈$840k/yr · news $67,767/window, ≈$1.167M model-grade scaled — both PAPER until the owner's gateway, both VALIDATED on every improvement axis anyone proposed) + **two forecast rails** (tape: HAR-RV gate · calendar: power layer + the collision-priced two-calendar layer) + **three never-gates observability surfaces** (nightly artifact · dashboard event_window tags · monitor --context). Laws: routing-never-fitting · information = $0 by design · state is telemetry · a protection trigger is never touched by an information change. The three-era verdict: **these workstreams bought certainty, not cashflow** — money not lost + the deployed edge armored + operational awareness. State of record: `SYSTEM-LAYERS-ANALYSIS.md` (incl. §5's answered profit question) + `PROGRESS-RECORD.md` (the quiet-green board). **This optimizer-arm issue is unchanged** — queue position stands.
